plv8_func.cc

```diff
@@ -186,15 +178,6 @@ SubTranBlock::exit(bool success)
 		ReleaseCurrentSubTransaction();
 	else
 		RollbackAndReleaseCurrentSubTransaction();
-
-	MemoryContextSwitchTo(m_mcontext);
-	CurrentResourceOwner = m_resowner;
-
-	/*
-	 * AtEOSubXact_SPI() should not have popped any SPI context, but just
-	 * in case it did, make sure we remain connected.
-	 */
-	SPI_restore_connection();
```

Comment:
In my testing, it was completely a no-op. `CurrentMemoryContext ` was not changed by `BeginInternalSubTransaction()` and reading the code for BeginInternalSubTransaction, I couldn't find any reason it would.

If there's something I am missing, can absolutely be put back. It seemed like no-op and felt hacky: if a sub transaction created a new memory context, why would we want to "go around" that?